pkg/getty/README.md

```diff
@@ -10,7 +10,7 @@ If you want a console getty, add the following to your `moby.yml`:
 ```
 services:
   - name: getty
-    image: "linuxkit/getty:e36b5e794256c2a56c000035f73a4d46f7ffb3e6"
```

Comment:
(wrote this in the wrong place so it was unthreaded, pasting it here, will delete the other comment)

`pkg/getty/README.md` is in `pkg/getty` and is therefore part of the hash, that's the problem I was getting at (note that it is the tree-hash of the directory, it is not a commit hash)

Moving some stuff to subdirs would be one was to resolve this, although in your proposal `Makefile` and `Dockerfile` would no longer be part of the hash but they should be because they impact the output.